services/logs2logs-db/pipeline/router-logs.conf

```diff
@@ -5,6 +5,11 @@ input {
     queue_size => 5000
     receive_buffer_bytes => 26214400
   }
+  http {
+    id => "http_input"
+    user => "${LOGSTASH_USERNAME:-username}"
```

Comment:
logsstash configs work different for not defined variables, see:
https://www.elastic.co/guide/en/logstash/current/environment-variables.html
`${var:default value}`
but actually, let's use the same system like we use  for `ELASTICSEARCH_URL`:
1. define  `LOGSTASH_USERNAME` and `LOGSTASH_PASSWORD` in the Dockerfile with `logstash-username` and `logstash-password`
2. then we override these env variables in realtime via the kubernetes yaml